### ISSUE TITLE
chemical washer not calculating correctly

### ISSUE THREAD

**Issue by Trixz2018** [mekanica/mekanica#151]
#### Issue description:
![2019-04-02_11 31 05](https://user-images.githubusercontent.com/8538100/55419647-cf8b7c80-553a-11e9-91b8-f1bc51a2c249.png)

chemical washer miscalculating which is causing the chemical crystallizer to become stuck

#### Steps to reproduce:
1.process lots of ore through chemical washer
2.
3.

#### Version (make sure you are on the latest version before reporting):
**Forge:** 
**Mekanica:** dev build
**Other relevant version:** 

#### If a (crash)log is relevant for this issue, link it here: (_It's almost always relevant_)

[gist/pastebin/etc link here]

**Comment by aidancbrady** [comment]
Are you sure it's the washer and not the connected machine miscalculating?
Also for completeness, are there any upgrades installed?

**Comment by Trixz2018** [comment]
yes there was full set of speed and energy upgrades on both the washer and the crystallizer, once i lowered the speed upgrades in the washer, it didnt seem to happen. At least for the couple of hours of playing after i did that. Which of course it could very well be the gas pipes as well, i didn't really consider them a factor.

**Comment by Trixz2018** [comment]
I will be editing this to add what ores causes the issue.

1. magnesium - Nuclearcraft
2. Cobalt - tinkers
